content/layering/layering.adoc

```diff
@@ -8,7 +8,31 @@
 :numbered:
 :homepage https://github.com/projectatomic/container-best-practices:
 
-Guidelines on creating layers.
+This chapter provides guidelines on creating layers.
+
+== Minimizing the Number of Layers
+
+To keep your Dockerfiles easily readable and maintainable in future, you should maintain a reasonable number of layers. In general, a new layer is created for every new instruction defined. However, this does not necessarily mean that one instruction should be associated with only one command or definition.
+To ensure transparency and good overview of the layer content, it is preferable to group related operations in one layer. For example, take this snippet from the https://github.com/openshift/sti-python/blob/master/3.3/Dockerfile[sti-python Dockerfile].
+
+----
+RUN yum install -y \
```

Comment:
for F22 we will be most likely switching to use `dnf`, but that is just a reminder :-)